sunbird-rc-esignet-integration-impl/README.md

```diff
@@ -1,44 +1,52 @@
-# mock-esignet-integration-impl
+# sunbird-rc-esignet-integration-impl
 
 ## About
 
-Implementation for all the interfaces defined in esignet-integration-api. This libaray is built as a wrapper for [mock-identity-system](mock-identity-system) service.
+Implementation for all the interfaces defined in esignet-integration-api. This libaray is built as a wrapper for [sunbird-registry-system](sunbird-registry-url) service.
 
 This library should be added as a runtime dependency to [esignet-service](https://github.com/mosip/esignet) for development purpose only.
 Note: This is not production use implementation.
```

Comment:
Above 2 lines are not true. This project is not just for development purpose, it can also be used for production uses.

Reply:
done